src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.wasm.cs

```diff
@@ -150,6 +151,13 @@ protected override Size ArrangeOverride(Size finalSize)
 			return base.ArrangeOverride(arrangeSize);
 		}
 
+		internal override void OnLayoutUpdated()
+		{
+			base.OnLayoutUpdated();
+
+			UpdateIsTextTrimmed();
```

Comment:
Let's not raise the event if no one is listening to it, it will degrade performance otherwise.

Reply:
Indeed, since OnLayoutUpdated would be a frequently triggered call. The problem however though, is that we need both: 1. updating a property; 2. firing an event on changed
for the first one, we have no way to predict if there will be an observer
unless we add a local flag on the TextBlock-level that the user can set, that block the call when unset

Reply:
Can't we assume that noone's using the property unless it has something else but the default value? Then we could trigger the computation when it's read once?

Reply:
you mean something like this?:
```cs
IsTextTrimmed.get => {
	if (!_accessed) { _accessed=true; UpdateIsTextTrimmed(); }
	return (bool)GetValue(IsTextTrimmedProperty)	
}
UpdateIsTextTrimmed() {
	if (!_accessed) return;
	// the rest ...
}
```